collectors/proc.plugin/sys_class_power_supply.c

```diff
@@ -215,6 +222,16 @@ int do_sys_class_power_supply(int update_every, usec_t dt) {
                                 pr->property_dim_root = pd;
                             }
                         }
+
+                        // create a zero empty/min dimension
+                        if(unlikely(max_value_found && !min_value_found)) {
```

Comment:
I'm not entirely sure that this should be marked as unlikely.  Most of the systems I've seen actually are missing at least one lower bound dimension (usually the `*_min_design` or `*_empty_design` dimensions).

I'm not entirely sure it _shouldn't_ be marked unlikely either though, given that it only really affects startup of the collector, and it shouldn't affect things much.